src/UrqlClient.re

```diff
@@ -320,6 +326,8 @@ let partialOpCtxToPartialOpCtxJs = opts =>
   switch (opts) {
   | Some(o) =>
     let url = o->ClientTypes.partialOpUrlGet;
+    // TODO: where does this "partialOpPollIntervalGet" come from...
+    let pollInterval = o->ClientTypes.partialOpPollIntervalGet;
```

Comment:
Good question! It gets autogenerated by the use of `[@bs.deriving abstract]`: https://bucklescript.github.io/docs/en/object#accessors Since `@bs.deriving abstract` generates an "abstract" type, whose internal structure is hidden from the outside world, the only typesafe way to access properties is via getters and setters that are auto-generated by BuckleScript. It's not the most ergonomic but it guarantees you can't access fields that don't exist using dot notation, i.e. `o.someOtherPollInterval`.